elastic/kibana
# [Uptime] Do not drop edge buckets

## Summary

Stop dropping incomplete buckets. ~Code is WIP~

We purposely started dropping the last bucket to avoid showing abnormally small chunks in our visualizations. This change seeks to avoid dropping buckets from our visualizations altogether.

Resolves https://github.com/elastic/uptime/issues/83

## Testing
0. Set up Heartbeat + ES + Kibana, ensure some monitors check at a relatively fast interval (~`@every 10s`)
1. Navigate to the Uptime solution
  a. Allow data to accumulate sufficiently that several buckets are displayed in the snapshot histogram
2. You should see your bucket filling up if you use the date picker's refresh button

### Checklist

Use ~~strikethroughs~~ to remove checklist items you don't feel are applicable to this PR.


### For maintainers
